SpywareFilter/sections/tracking_servers_firstparty.txt

```diff
@@ -1510,3 +1510,6 @@ charlestownwyllie.oaklawnnonantum.com^
 ||log2.musical.ly^
 ||frontier.musical.ly^
 ||xlog-va.musical.ly^
+! Symantec / Norton
+||stats.norton.com^
+||oms.symantec.com^
```

Comment:
It seems this domain cannot be blocked
https://groups.google.com/forum/#!topic/alt.comp.anti-virus/GmIJWpmg7ho